api-specificatie/openapi.yaml

```diff
@@ -1277,14 +1307,14 @@ components:
           minLength: 1
           maxLength: 40
           example: 'AV-2019-08-19-00'
-        ligtInWoonplaats:
-          $ref: '#/components/schemas/NEN3610ID'
+        woonplaatsIdentificatie:
```

Comment:
Soms worden gerelateerde objecten met property namen <object type>Identificatie en soms met <object type> aangeduid. Graag consistent maken.

Reply:
De afspraak die we gemaakt hebben is dat de propertynamen in de resource zel waar de identificatie(s) van de gerelateerde resource zijn opgenomen worden opgesteld als resourcenaam + _identificatie_  en in meervoudsvorm als dat van toepassing is. 

In de _links en de _embedded worden de namen van de _resources_ gebruikt waarnaar verwezen wordt. 

Dit is volgens mij nu consquent doorgeveord, maar als ik dat ergens niet consequent heb gedaan hoor ik dat graag.